tools/run_tests/xds_k8s_test_driver/README.md

```diff
@@ -174,7 +174,13 @@ export KUBE_CONTEXT="$(kubectl config current-context)"
 
 ```shell
 # Create python virtual environment
-python3.7 -m venv venv
+python3 -m venv venv
+
+# On some Ubuntu systems previous command may fail with the message saying:
```

Comment:
Let's add this to the ("Requirements")[https://github.com/grpc/grpc/blob/master/tools/run_tests/xds_k8s_test_driver/README.md#requirements] section instead. Just list it below `Python v3.7+`.

Reply:
* it = [python3-venv](https://docs.python.org/3.7/library/venv.html).

Reply:
Thanks for your review. I reorganized the change. PTAL.